tools/serverpod_cli/bin/create/copier.dart

```diff
@@ -40,7 +40,7 @@ class Copier {
       }
       if (entity is Directory) {
         var dirName = p.basename(entity.path);
-        _copyDirectory(entity, '$relativePath$dirName/');
+        _copyDirectory(entity, p.join(relativePath, dirName));
```

Comment:
Has this been tested on MacOS? It removes the ending slash, which may cause issues with the `cp` command.

Reply:
Yes it does work

<details>

  <summary> Logs </summary>

```console
minnu@Minnus-Mac ~ % /Users/minnu/Desktop/serverpod/tools/serverpod_cli/scm create -f dummy
Creating project dummy...
  /Users/minnu/dummy/dummy_server/generated/tables-serverpod.pgsql
  /Users/minnu/dummy/dummy_server/generated/protocol.yaml
  /Users/minnu/dummy/dummy_server/generated/tables.pgsql
  /Users/minnu/dummy/dummy_server/docker-compose.yaml
  /Users/minnu/dummy/dummy_server/bin/main.dart
  /Users/minnu/dummy/dummy_server/CHANGELOG.md
  /Users/minnu/dummy/dummy_server/config/generator.yaml
  /Users/minnu/dummy/dummy_server/config/staging.yaml
  /Users/minnu/dummy/dummy_server/config/production.yaml
  /Users/minnu/dummy/dummy_server/config/development.yaml
  /Users/minnu/dummy/dummy_server/config/passwords.yaml
  /Users/minnu/dummy/dummy_server/README.md
  /Users/minnu/dummy/dummy_server/pubspec.yaml
  /Users/minnu/dummy/dummy_server/setup-tables
  /Users/minnu/dummy/dummy_server/lib/server.dart
  /Users/minnu/dummy/dummy_server/lib/src/generated/endpoints.dart
  /Users/minnu/dummy/dummy_server/lib/src/generated/protocol.dart
  /Users/minnu/dummy/dummy_server/lib/src/generated/example_class.dart
  /Users/minnu/dummy/dummy_server/lib/src/endpoints/example_endpoint.dart
  /Users/minnu/dummy/dummy_server/lib/src/future_calls/example_future_call.dart
  /Users/minnu/dummy/dummy_server/lib/src/protocol/example_class.yaml
  /Users/minnu/dummy/dummy_server/analysis_options.yaml
  /Users/minnu/dummy/dummy_server/gitignore
  /Users/minnu/dummy/dummy_client/CHANGELOG.md
  /Users/minnu/dummy/dummy_client/README.md
  /Users/minnu/dummy/dummy_client/pubspec.yaml
  /Users/minnu/dummy/dummy_client/lib/PROJECTNAME_client.dart
  /Users/minnu/dummy/dummy_client/lib/src/protocol/client.dart
  /Users/minnu/dummy/dummy_client/lib/src/protocol/protocol.dart
  /Users/minnu/dummy/dummy_client/lib/src/protocol/example_class.dart
  /Users/minnu/dummy/dummy_client/analysis_options.yaml
  /Users/minnu/dummy/dummy_client/gitignore
  /Users/minnu/dummy/dummy_flutter/test/widget_test.dart
  /Users/minnu/dummy/dummy_flutter/README.md
  /Users/minnu/dummy/dummy_flutter/pubspec.yaml
  /Users/minnu/dummy/dummy_flutter/lib/main.dart
  /Users/minnu/dummy/dummy_flutter/analysis_options.yaml
  /Users/minnu/dummy/dummy_flutter/gitignore

Running `dart pub get` in /Users/minnu/dummy/dummy_server
Resolving dependencies...
+ args 2.3.0
+ async 2.9.0
+ buffer 1.1.1
+ charcode 1.3.1
+ collection 1.16.0
+ crypto 3.0.2
+ executor 2.2.2
+ http 0.13.4
+ http_parser 4.0.0
+ lints 1.0.1 (2.0.0 available)
+ meta 1.7.0
+ path 1.8.1
+ pedantic 1.11.1
+ postgres 2.4.3
+ redis 2.2.0 (3.0.0 available)
+ retry 3.1.0
+ sasl_scram 0.1.0
+ saslprep 1.0.2
+ serverpod 0.9.6
+ serverpod_client 0.9.6
+ serverpod_postgres_pool 2.1.1
+ serverpod_serialization 0.9.6
+ serverpod_service_client 0.9.6
+ serverpod_shared 0.9.6
+ source_span 1.9.0
+ stack_trace 1.10.0
+ stream_channel 2.1.0
+ string_scanner 1.1.0
+ term_glyph 1.2.0
+ typed_data 1.3.0
+ unorm_dart 0.2.0
+ vm_service 8.2.2
+ web_socket_channel 2.2.0
+ yaml 3.1.0
Changed 34 dependencies!

Running `dart pub get` in /Users/minnu/dummy/dummy_client
Resolving dependencies...
+ async 2.9.0
+ charcode 1.3.1
+ collection 1.16.0
+ crypto 3.0.2
+ http 0.13.4
+ http_parser 4.0.0
+ meta 1.7.0
+ path 1.8.1
+ serverpod_client 0.9.6
+ serverpod_serialization 0.9.6
+ source_span 1.9.0
+ stream_channel 2.1.0
+ string_scanner 1.1.0
+ term_glyph 1.2.0
+ typed_data 1.3.0
+ web_socket_channel 2.2.0
+ yaml 3.1.0
Changed 17 dependencies!

Running `flutter create .` in /Users/minnu/dummy/dummy_flutter
Signing iOS app for device deployment using developer identity: "Apple Development: hi2minnu@gmail.com (B2AWC6GV75)"
Recreating project ....
  windows/runner/flutter_window.cpp (created)
  windows/runner/utils.h (created)
  windows/runner/utils.cpp (created)
  windows/runner/runner.exe.manifest (created)
  windows/runner/CMakeLists.txt (created)
  windows/runner/win32_window.h (created)
  windows/runner/Runner.rc (created)
  windows/runner/win32_window.cpp (created)
  windows/runner/resources/app_icon.ico (created)
  windows/runner/main.cpp (created)
  windows/runner/resource.h (created)
  windows/runner/flutter_window.h (created)
  windows/flutter/CMakeLists.txt (created)
  windows/.gitignore (created)
  windows/CMakeLists.txt (created)
  ios/Runner.xcworkspace/contents.xcworkspacedata (created)
  ios/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (created)
  ios/Runner/Info.plist (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage@2x.png (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage@3x.png (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/Contents.json (created)
  ios/Runner/Assets.xcassets/LaunchImage.imageset/LaunchImage.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-76x76@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-1024x1024@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-83.5x83.5@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-20x20@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60@2x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-76x76@1x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-40x40@3x.png (created)
  ios/Runner/Assets.xcassets/AppIcon.appiconset/Icon-App-29x29@2x.png (created)
  ios/Runner/Base.lproj/LaunchScreen.storyboard (created)
  ios/Runner/Base.lproj/Main.storyboard (created)
  ios/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata (created)
  ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  ios/Runner.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings (created)
  ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme (created)
  ios/Flutter/Debug.xcconfig (created)
  ios/Flutter/Release.xcconfig (created)
  ios/Flutter/AppFrameworkInfo.plist (created)
  ios/.gitignore (created)
  dummy_flutter.iml (created)
  web/favicon.png (created)
  web/index.html (created)
  web/manifest.json (created)
  web/icons/Icon-maskable-512.png (created)
  web/icons/Icon-192.png (created)
  web/icons/Icon-maskable-192.png (created)
  web/icons/Icon-512.png (created)
  .metadata (created)
  macos/Runner.xcworkspace/contents.xcworkspacedata (created)
  macos/Runner.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_16.png (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_1024.png (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_256.png (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_64.png (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_512.png (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_128.png (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/Contents.json (created)
  macos/Runner/Assets.xcassets/AppIcon.appiconset/app_icon_32.png (created)
  macos/Runner/DebugProfile.entitlements (created)
  macos/Runner/Base.lproj/MainMenu.xib (created)
  macos/Runner/MainFlutterWindow.swift (created)
  macos/Runner/Configs/Debug.xcconfig (created)
  macos/Runner/Configs/Release.xcconfig (created)
  macos/Runner/Configs/Warnings.xcconfig (created)
  macos/Runner/Configs/AppInfo.xcconfig (created)
  macos/Runner/AppDelegate.swift (created)
  macos/Runner/Info.plist (created)
  macos/Runner/Release.entitlements (created)
  macos/Runner.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist (created)
  macos/Runner.xcodeproj/project.pbxproj (created)
  macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme (created)
  macos/Flutter/Flutter-Debug.xcconfig (created)
  macos/Flutter/Flutter-Release.xcconfig (created)
  macos/.gitignore (created)
  android/app/src/profile/AndroidManifest.xml (created)
  android/app/src/main/res/mipmap-mdpi/ic_launcher.png (created)
  android/app/src/main/res/mipmap-hdpi/ic_launcher.png (created)
  android/app/src/main/res/drawable/launch_background.xml (created)
  android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png (created)
  android/app/src/main/res/mipmap-xxhdpi/ic_launcher.png (created)
  android/app/src/main/res/values-night/styles.xml (created)
  android/app/src/main/res/values/styles.xml (created)
  android/app/src/main/res/drawable-v21/launch_background.xml (created)
  android/app/src/main/res/mipmap-xhdpi/ic_launcher.png (created)
  android/app/src/main/AndroidManifest.xml (created)
  android/app/src/debug/AndroidManifest.xml (created)
  android/gradle/wrapper/gradle-wrapper.properties (created)
  android/gradle.properties (created)
  android/.gitignore (created)
  android/settings.gradle (created)
  android/app/build.gradle (created)
  android/app/src/main/kotlin/com/example/dummy_flutter/MainActivity.kt (created)
  android/build.gradle (created)
  android/dummy_flutter_android.iml (created)
  ios/Runner/Runner-Bridging-Header.h (created)
  ios/Runner/AppDelegate.swift (created)
  ios/Runner.xcodeproj/project.pbxproj (created)
  .idea/runConfigurations/main_dart.xml (created)
  .idea/libraries/Dart_SDK.xml (created)
  .idea/libraries/KotlinJavaRuntime.xml (created)
  .idea/modules.xml (created)
  .idea/workspace.xml (created)
Running "flutter pub get" in dummy_flutter...                    2,974ms
Wrote 116 files.

All done!
In order to run your application, type:

  $ cd .
  $ flutter run

Your application code is in ./lib/main.dart.


Setting up Docker and default database tables in /Users/minnu/dummy/dummy_server
If you run serverpod create for the first time, this can take a few minutes as Docker is downloading the images for Postgres. If you get stuck at this step,
make sure that you have the latest version of Docker Desktop and that it is currently running.

Starting docker
Creating network "dummy_server_default" with the default driver
Creating dummy_server_redis_1 ... 
Creating dummy_server_postgres_1 ... 
Creating dummy_server_postgres_1 ... done
Creating dummy_server_redis_1    ... done
Postgres is ready
Creating dummy_server_postgres_run ... 
Creating dummy_server_postgres_run ... done
psql: error: connection to server at "postgres" (172.19.0.3), port 5432 failed: FATAL:  password authentication failed for user "postgres"
2
Stopping docker
Stopping dummy_server_redis_1    ... 
Stopping dummy_server_postgres_1 ... 
Stopping dummy_server_postgres_1 ... done
Stopping dummy_server_redis_1    ... done
Completed table setup exit code: 0
Cleaning up



=== SERVERPOD CREATED ===

All setup. You are ready to rock!

Start your Serverpod server by running:

  $ cd dummy/dummy_server
  $ serverpod run

You can also start Serverpod manually by running:

  $ cd dummy/dummy_server
  $ docker-compose up --build --detach
  $ dart bin/main.dart

```

</details>